.github/workflows/build-beta.yml

```diff
@@ -35,10 +35,9 @@ jobs:
 
       - name: Flag Non-Experimental Build
         run: |
-          sed -i -e "s/--@experimental@/--\[=====\[@experimental@/" WeakAuras/Init.lua \
-          -e "s/--@end-experimental@/--@end-experimental\]=====\]/" WeakAuras/Init.lua \
-          -e "s/--\[=====\[@non-experimental@/--@non-experimental@/" WeakAuras/Init.lua \
-          -e "s/--@end-non-experimental@\]=====\]/--@end-non-experimental/" WeakAuras/Init.lua
+          sed -i \
+          -e "s/--@experimental@/--\[=====\[@experimental@/" \
+          -e "s/--@end-experimental@/--@end-non-experimental@\]=====\]/" WeakAuras/Init.lua
```

Comment:
There's an inconsequential typo in the replacement string, should be --@end-experimental@\]=====\]